include/SDL3/SDL_begin_code.h

```diff
@@ -225,3 +225,14 @@
 #define SDL_ALLOC_SIZE2(p1, p2)
 #endif
 #endif /* SDL_ALLOC_SIZE2 not defined */
+
+#ifndef SDL_RESTRICT
+#if defined(__GNUC__)
+#define SDL_RESTRICT __restrict__
+#elif defined(_MSC_VER)
+#define SDL_RESTRICT __restrict
+#else
+#warning "RESTRICTED unsupported"
```

Comment:
We shouldn't have a warning like this in public headers. This isn't a required feature, and we don't want noise when building other people's projects.

Reply:
Oops. Removed it.